server/test/service/ScalaTestRunnerTest.scala

```diff
@@ -1,10 +1,10 @@
 package service
 
 import org.scalatest.{FlatSpec, Matchers, Suite}
+import service.ScalaTestRunnerTest._
 
 class ScalaTestRunnerTest extends FlatSpec with Matchers {
   behavior of "ScalaTestRunner"
-  val correctSolution = "class A { def sleepIn(weekday: Boolean, vacation: Boolean): Boolean = {!weekday || vacation}}"
```

Comment:
What is the reason to have val correctSolution in companion object, but incorrect solution val in class members?